### ISSUE TITLE
setConfigXML returns 500 Internal Server Error

### ISSUE THREAD

**Issue by goldsky** [bigbluebutton/bigbluebutton-api-php#6]
I'm trying to use `setConfigXML` API, but it returns 500 Internal Server Error.

``` html
<!DOCTYPE html>
<!--[if lt IE 7 ]>
<html lang="en" class="no-js ie6">
    <![endif]-->
    <!--[if IE 7 ]>
    <html lang="en" class="no-js ie7">
        <![endif]-->
        <!--[if IE 8 ]>
        <html lang="en" class="no-js ie8">
            <![endif]-->
            <!--[if IE 9 ]>
            <html lang="en" class="no-js ie9">
                <![endif]-->
                <!--[if (gt IE 9)|!(IE)]>
                <!-->
                <html lang="en" class="no-js">
                    <!--
                    <![endif]-->
                    <head>
                        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
                        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
                        <title>Error</title>
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <link rel="shortcut icon" href="/bigbluebutton/static/images/favicon.ico" type="image/x-icon">
                        <link rel="apple-touch-icon" href="/bigbluebutton/static/images/apple-touch-icon.png">
                        <link rel="apple-touch-icon" sizes="114x114" href="/bigbluebutton/static/images/apple-touch-icon-retina.png">
                        <link rel="stylesheet" href="/bigbluebutton/static/css/main.css" type="text/css">
                        <link rel="stylesheet" href="/bigbluebutton/static/css/mobile.css" type="text/css">
                        <meta name="layout" content="main">




                    </head>
                    <body>
                        <div id="grailsLogo" role="banner">
                            <a href="http://grails.org">
                                <img src="/bigbluebutton/static/images/grails_logo.png" alt="Grails"/>
                            </a>
                        </div>
                        <ul class="errors">
                            <li>An error has occurred</li>

                        </ul>
                        <div class="footer" role="contentinfo"></div>
                        <div id="spinner" class="spinner" style="display:none;">Loading&hellip;</div>
                        <script src="/bigbluebutton/static/js/application.js" type="text/javascript" ></script>


                    </body>
                </html>
```

[What I've set were](https://github.com/virtudraft/bbbx/blob/master/core%2Fcomponents%2Fbbbx%2Fmodel%2Fbigbluebutton.class.php#L211):
1. set the URL up: meetingID=...&configXML=...&checksum=... 
2. content-type to `application/x-www-form-urlencoded`
3. post field body as same as the URL ([http_build_query($postFields)](https://github.com/virtudraft/bbbx/blob/master/core%2Fcomponents%2Fbbbx%2Fmodel%2Fbigbluebutton.class.php#L351)).

Is there something I missed?
Thanks ahead!

**Comment by GhaziTriki** [comment]
I don't remember that```setConfigXML``` have been implemented yet in that library.
However can you please check http://docs.bigbluebutton.org/dev/api.html#setconfigxml and tell if you still have the same issue?

**Comment by GhaziTriki** [comment]
```setConfigXML``` will be implemented very soon. I will get back to you once implemented.

**Comment by GhaziTriki** [comment]
It is now implemented in PR #18
